maps/sccv_horizon/sccv_horizon.dmm

```diff
@@ -50306,6 +50306,15 @@
 	pixel_y = 4;
 	req_one_access = list(38,19,72)
 	},
+/obj/machinery/ringer/north{
+	dir = 8;
+	pixel_y = 0;
+	pixel_x = -10;
+	department = "Consular A";
+	id = "consular_a_ringer";
+	req_access = null;
+	req_one_access = list(72)
```

Comment:
Make subtypes instead of setting this in map

Reply:
None of the ringers are subtypes.